content/en/docs/staging-environment.md

```diff
@@ -29,15 +29,32 @@ The staging environment uses the same rate limits as [described for the producti
+If you wish to modify a test-only client to trust the staging environment for testing purposes you can do so by adding their certificates to your testing trust store. **Important note:** Do not add the staging root or intermediate to a trust store that you use for ordinary browsing or other activities, since they are not audited or held to the same standards as our production roots, and so are not safe to use for anything other than testing.
 
-## Root Certificates
+* **Pretend Pear X1**
+  * Subject: `O = (STAGING) Internet Security Research Group, CN = (STAGING) Pretend Pear X1`
+  * Key type: `RSA 4096`
+  * Certificate details: [der](/certs/staging/letsencrypt-stg-root-x1.der), [pem](/certs/staging/letsencrypt-stg-root-x1.pem), [txt](/certs/staging/letsencrypt-stg-root-x1.txt)
+* **Bogus Broccoli X2**
+  * Subject: `O = (STAGING) Internet Security Research Group, CN = (STAGING) Bogus Broccoli X2`
+  * Key type: `ECDSA P-384`
+  * Certificate details (self-signed): [der](/certs/staging/letsencrypt-stg-root-x2.der), [pem](/certs/staging/letsencrypt-stg-root-x2.pem), [txt](/certs/staging/letsencrypt-stg-root-x2.txt)
+  * Certificate details (cross-signed by Pretend Pear X1): [der](/certs/staging/letsencrypt-stg-root-x2-signed-by-x1.der), [pem](/certs/staging/letsencrypt-stg-root-x2-signed-by-x1.pem), [txt](/certs/staging/letsencrypt-stg-root-x2-signed-by-x1.txt)
 
-The staging environment has two active root certificates which are **not present** in browser/client trust stores: "(STAGING) Pretend Pear X1" and "(STAGING) Bogus Broccoli X2". If you wish to modify a test-only client to trust the staging environment for testing purposes you can do so by adding the ["(STAGING) Pretend Pear X1"](/certs/staging/letsencrypt-stg-root-x1.pem) and/or ["(STAGING) Bogus Broccoli X2"](/certs/staging/letsencrypt-stg-root-x2.pem) certificate to your testing trust store. You can find all of our staging certificates [here](https://github.com/letsencrypt/website/tree/master/static/certs/staging).  Important: Do not add the staging root or intermediate to a trust store that you use for ordinary browsing or other activities, since they are not audited or held to the same standards as our production roots, and so are not safe to use for anything other than testing.
+## Subordinate (Intermediate) CAs
+
+The staging environment has four active intermediate certificates, issued from the untrusted roots detailed above. Currently, these are:
```

Comment:
I think we should list the inactive entries too (we're including them in static/ after all)